jekyll/_cci2_ja/settings.md

```diff
@@ -10,7 +10,7 @@ order: 2
 
 CircleCIドキュメントのこの設定セクションに含まれるガイドでは、CircleCI アプリケーションのプロジェクト設定と組織設定のページからアクセスできる機能とタスクについて説明します。 以下のトピックが含まれています。
 
-* [VCS integration]({{ site.baseurl }}/ja/gh-bb-integration/) (GitHub/Bitbucket): アクセス許可とキーに関する情報
+* [VCS integration]({{ site.baseurl }}/ja/github-integration/) (GitHub/Bitbucket): アクセス許可とキーに関する情報
```

Comment:
@rosieyohannan I have done this replacement on my own. Can you please make sure this is accurate? thanks